frame/support/src/lib.rs

```diff
@@ -66,6 +66,8 @@ pub mod inherent;
 pub mod error;
 pub mod instances;
 pub mod migrations;
+mod paritydb_weights;
+mod rocksdb_weights;
```

Comment:
They should be below the `weights` module.

Reply:
updated

Reply:
Looks like `fmt` is not happy with this: https://gitlab.parity.io/parity/substrate/-/jobs/1399806#L89  
I could add a newline, then it looks like this after `fmt`:  
```rust
...
pub mod traits;

mod paritydb_weights;
mod rocksdb_weights;
pub mod weights;
```

Reply:
Okay works now